app/src/main/res/layout/partial_itunes_row.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/textView2"
```

Comment:
Can we name this better?

Starting the id with `textView` is good. Let's try and have a more descriptive name.